app/uk/gov/hmrc/bindingtariffclassification/repository/CaseRepository.scala

```diff
@@ -45,13 +46,42 @@ trait CaseRepository {
 
   def getByReference(reference: String): Future[Option[Case]]
 
-  def get(searchBy: CaseParamsFilter, sortedBy: Option[CaseSort]): Future[Seq[Case]]
+  def get(search: Search): Future[Seq[Case]]
 
   def deleteAll(): Future[Unit]
 }
 
 @Singleton
-class CaseMongoRepository @Inject()(mongoDbProvider: MongoDbProvider, jsonMapper: JsonObjectMapper)
+class EncryptingMongoRepository @Inject()(repository: CaseMongoRepository, crypto: Crypto, appConfig: AppConfig) extends CaseRepository {
```

Comment:
We have now two mongo concrete singleton implementations of the trait `CaseRepository`.
How do you tell the Play microservice that we want only one to run? 
I think we do not want both running